cumulusci/tasks/apex/testrunner.py

```diff
@@ -115,23 +112,24 @@ class RunApexTests(BaseSalesforceApiTask):
         },
         "poll_interval": {
             "description": (
-                "Seconds to wait between polling for Apex test "
-                + "results.  Defaults to 3"
+                "Seconds to wait between polling for Apex test " + "results."
             )
         },
-        "retries": {"description": "Number of retries (default=10)"},
-        "retry_interval": {
-            "description": "Number of seconds to wait before the next retry (default=5),"
-        },
-        "retry_interval_add": {
-            "description": "Number of seconds to add before each retry (default=5),"
-        },
```

Comment:
Are we sure that no one is using the old retry mechanism?

Reply:
Some quick searching suggests no at least within our products. We should probably mention the removal of this in the changenotes though.

Reply:
Good point. I removed it on the basis that it was retrying something that should never fail in a retryable manner - a single Tooling API query. That query should either return no results, or it should fail with some kind of authentication problem that's not retryable (retriable?).

Reply:
Yeah, I agree with removing it.

Reply:
Added to change list.